test/worker/python/single-target-changesoundeffectby.py

```diff
@@ -0,0 +1,6 @@
+## -- target1 -- ##
+
+def target1_0(vm_proxy):
```

Comment:
```suggestion
async def thread_id_0(vm_proxy):
```

Input python function headers need to be formatted like this to fit new linker specifications